### PR TITLE
feat(@vtmn/svelte): add `VtmnDropdown` component

### PR DESCRIPTION
**Pull request by Tlahey**
Refer to #1051 
Based on the work of @jb-thery 

## Changes description

This P.R adds the Svelte dropdown component

## Context

Related issue [#364](https://github.com/Decathlon/vitamin-web/issues/364)

## Checklist

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Please, don't request directly from your main!
- [x] Check commits & PR names matches our requested structure. It must follow the https://www.conventionalcommits.org pattern.
- [x] Check your code additions will fail neither code linting checks.
- [x] I have reviewed the submitted code.
- [x] I have tested on related showcases.
- [x] If it includes design changes, please ask for a review with a core team designer.

## Does this introduce a breaking change?

- No

## Other information

Screenshot 2022-03-17 at 20 08 46

### CODE REVIEW

File: packages/showcases/core/csf/components/actions/dropdown.csf.js [hunk 1]
```diff
@@ -1,9 +1,54 @@
...
+    description: 'Disabled state',
+    defaultValue: false,
+    control: {
+      type: 'boolean',
+    },
+  },
+
+  menuMaxHeight: {
+    type: { name: 'number', required: false },
+    description: 'Max height of menu, this enable scroll',
```
Comment (GaspardMathon) [inline packages/showcases/core/csf/components/actions/dropdown.csf.js:48]:
```suggestion
    description: 'Max height of menu, this enables scroll',
```

**Review by GaspardMathon — APPROVED** [review]

**Comment by Tlahey** [comment]
Need to update code, refer A11y -> #1113

**Comment by lauthieb** [comment]
> Need to update code, refer A11y -> #1113

What do you mean @Tlahey by this comment?

**Comment by lauthieb** [comment]
@Tlahey thanks! Is it ok for you? If yes we will take this PR ready for review again :)

**Comment by Tlahey** [comment]
@lauthieb yes it's good for me :)

**Review by GaspardMathon — DISMISSED** [review]

File: packages/showcases/core/csf/components/actions/dropdown.csf.js [hunk 2]
```diff
@@ -1,9 +1,54 @@
...
+  },
+
+  menuMaxHeight: {
+    type: { name: 'number', required: false },
+    description: 'Max height of menu, this enables scroll',
+    defaultValue: 0,
+    control: {
+      type: 'number',
+    },
+  },
```
Comment (lauthieb) [inline packages/showcases/core/csf/components/actions/dropdown.csf.js:53]: This intelligence should be placed on CSS part instead.

**Review by lauthieb — CHANGES_REQUESTED** [review]
Thanks @Tlahey

File: packages/showcases/svelte/stories/components/actions/VtmnDropdown/VtmnDropdown.stories.svelte [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+      <VtmnDropdownItem
+        id={`with-max-height-opt-${index}`}
+        bind:group={groupWithDivider}
+        value={index}
+        icon="user-line"
+        divider={index !== 19}>Dropdown Option {index}</VtmnDropdownItem
+      >
+    {/each}
+  </VtmnDropdown>
+</Story>
```
Comment (lauthieb) [inline packages/showcases/svelte/stories/components/actions/VtmnDropdown/VtmnDropdown.stories.svelte:75]: Same here

**Review by lauthieb — CHANGES_REQUESTED** [review]

File: packages/sources/svelte/src/components/actions/VtmnDropdownItem/VtmnDropdownItem.svelte [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+  /**
+   * @type {object[]} Array of selected items
+   * @requires
+   */
+  export let group;
+
+  /**
+   * @type {boolean} If the dropdown item are checked
+   */
+  export let checked = false;
```
Comment (lauthieb) [inline packages/sources/svelte/src/components/actions/VtmnDropdownItem/VtmnDropdownItem.svelte:27]: should be renamed to "selected" has we did in design side.

**Review by lauthieb — CHANGES_REQUESTED** [review]

**Comment by Tlahey** [comment]
FYI : add max-height 260px to fit 5 items without scroll on the dropdown

**Review by lauthieb — APPROVED** [review]

**Comment by lauthieb** [comment]
@jb-thery & @Tlahey , AWESOME!! Thanks :tada:

**Pull request merged**